plugins/UM3NetworkPrinting/src/Cloud/Models/CloudClusterResponse.py

```diff
@@ -16,13 +16,14 @@ class CloudClusterResponse(BaseCloudModel):
     #  \param status: The status of the cluster authentication (active or inactive).
     #  \param host_version: The firmware version of the cluster host. This is where the Stardust client is running on.
     def __init__(self, cluster_id: str, host_guid: str, host_name: str, is_online: bool, status: str,
-                 host_version: Optional[str] = None, **kwargs) -> None:
+                 host_internal_ip: Optional[str], host_version: Optional[str] = None, **kwargs) -> None:
```

Comment:
The `Optional` here is for Mypy and that is fine. It means you can pass in `None` as a value. But if you want people to be able to call this constructor without passing any value for `host_internal_ip`, then you need to set a default value.

I think the unit tests may be failing because of this.